src/models/team.js

```diff
@@ -75,37 +75,53 @@ async function resolveMemberNames(ids) {
     }))
   }
 
-  try {
-    const resp = await request(
-      join(serverRuntimeConfig.OSM_API, `/api/0.6/users?users=${ids.join(',')}`)
-    )
-    var parser = new xml2js.Parser()
-
-    return new Promise((resolve, reject) => {
-      parser.parseString(resp, (err, xml) => {
-        if (err) {
-          reject(err)
-        }
+  // get the display names from the database table first
```

Comment:
@kamicut it would be nice to add test coverage for this. One possible approach is to mock on Cypress the OSM route `/api/0.6/users?users=` in a team page and check if the front end is populating the tables with usernames generated by the mock route.

Reply:
I've added an API test for this